process/modules/ylre_katualueet.py

```diff
@@ -36,7 +36,7 @@ def purpose_to_class(purpose: str) -> str:
         )
         df = df[~df["ylre_class"].isna()]
         df = df[~(df["geometry"].isna() | df["geometry"].is_empty)].loc[
-            :, ["ylre_class", "geometry"]
+            :, ["ylre_class", "kadun_nimi", "geometry"]
```

Comment:
Is this related to HAI-2416?

Reply:
No. Need to add "kadun_nimi" attribute for ylre_katualueet because it is used for to get that information for cycle_infra.